x-pack/plugins/licensing/README.md

```diff
@@ -0,0 +1,94 @@
+# Licensing plugin
+
+Retrieves license data from Elasticsearch and becomes a source of license data for all Kibana plugins on server-side and client-side.
+
+## API: 
+### Server-side
```

Comment:
I hope we are able to generate docs for all plugins soon https://github.com/elastic/kibana/issues/51736